### PR TITLE
Update ProxySQL types and helpers

### PR DESCRIPTION
**Pull request by TasdidurRahman**
Signed-off-by: TasdidurRahman <tasdid@appscode.com>

### CODE REVIEW

File: apis/kubedb/v1alpha2/proxysql_types.go [hunk 1]
```diff
@@ -102,6 +102,7 @@ type ProxySQLCertificateAlias string
 
 const (
 	ProxySQLServerCert          ProxySQLCertificateAlias = "server"
+	ProxySQLClientCert          ProxySQLCertificateAlias = "client"
 	ProxySQLArchiverCert        ProxySQLCertificateAlias = "archiver"
```
Comment (spectro30) [inline apis/kubedb/v1alpha2/proxysql_types.go:106]: remove `ProxySQLArchiverCert`.

**Review by spectro30 — CHANGES_REQUESTED** [review]

File: apis/kubedb/v1alpha2/proxysql_types.go (hunk 1 above)
Comment (TasdidurRahman) [reply apis/kubedb/v1alpha2/proxysql_types.go]: done

**Review by spectro30 — DISMISSED** [review]
LGTM

**Review by spectro30 — APPROVED** [review]
LGTM

**Pull request merged**